[BUG] Update description for 1102 in the 7.15 RN (#1082)
elastic/security-docs: docs/release-notes/7.15.asciidoc

[discrete]
[[breaking-changes-7.15.0]]
==== Breaking changes

> * After upgrading to {stack} version 7.15.x from a release between 7.12.0 and 7.14.2, you need to migrate detection alerts enriched with threat intelligence data to ensure threat intelligence properly displays in {elastic-sec}. For more information, refer to <<post-upgrade-req-cti-alerts, instructions for migrating detection alerts enriched with threat intelligence data>> ({pull}1102[#1102]).

Reviewer: Does the range _include_ the versions specified here (>=7.12.0 and <=7.14.2), or is it literally _between_ the two versions (>7.12.0 and <7.14.2)? I'm guessing the former, so this might be clearer:
```suggestion
* After upgrading to {stack} version 7.15.x from release versions 7.12.0 through 7.14.2, you need to migrate detection alerts enriched with threat intelligence data to ensure threat intelligence properly displays in {elastic-sec}. For more information, refer to <<post-upgrade-req-cti-alerts, instructions for migrating detection alerts enriched with threat intelligence data>> ({pull}1102[#1102]).
```

Author: It's the former and agree that "through" is more clear than "between" in this situation.
